content/blog/posts/o3de-motion-matching-2022-04.md

```diff
@@ -0,0 +1,179 @@
+---
+title: "Motion Matching in O3DE, a Data-Driven Animation Technique"
+date: 2022-05-12
+slug: blog-motionmatching
+author: Benjamin Jillich
+blog_img: "/images/blog/motion-matching/blog_img.png"
+full_img: "/images/blog/motion-matching/full_image.png"
+---
+
+Motion matching is a data-driven animation technique that is gaining popularity since games like 'The Last Of Us Part II', 'FIFA', and 'Half-Life: Alyx' have adotped it. The complexity of animation graphs has increased along with higher expectations, a greater variety of interactions between characters and their environments, and the need for more realistic animations. Motion matching is a way to animate a character in a controlled way by jumping and blending between pre-recorded animation sequences several times a second, synthesizing the animation for the character. It can be used for realistic locomotion without the need to manually create carefully timed transition points and synchronization between animation clips.
```

Comment:
IMHO the second sentance lacks a little clarity " The complexity of animation graphs has increased along with higher expectations, a greater variety of interactions between characters and their environments, and the need for more realistic animations."

Maybe something along the lines of "[Users|Players|Gamers] have higher expectations for animation quality including more realistic animations and a greater variety of interactions between characters and their environments. This growth in expectation is matched by a growth in animation graph complexity which has made [balancing|generating|developing?] them more expensive.

nit: Is "locomotion" the right term in the last sentence? Would argue that not all anim graphs involve motion that moves a character from one place to another.

Reply:
Changed it to:

"Users have higher expectations for animation quality including more realistic animations and a greater variety of interactions between characters and their environments. This growth in expectation resulted in increasingly complex animation graphs that became harder to maintain and more expensive to create."

Does that sound ok?

Reply:
Sounds great